### PR TITLE
Create proc_creation_win_susp_secedit.yml

### PR DESCRIPTION
**Pull request by blueteam0ps**
This was created based off my research https://blueteamops.medium.com/secedit-and-i-know-it-595056dee53d

### CODE REVIEW

File: rules/windows/process_creation/proc_creation_win_susp_secedit.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    selection_configure:
+        CommandLine|contains|all:
+            - '/configure'
+            - '/db'
+    filter:
+        - SubjectUserName|endswith: '$'
+    condition: process_name and (1 of selection*) and not filter
+falsepositives:
+    - Legitimate administrative use
+level: medium
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_susp_secedit.yml:49]:
```suggestion
logsource:
    category: process_creation
    product: windows
detection:   
    selection_img:
        - Image|endswith: '\secedit.exe'
        - OriginalFileName: 'SeCEdit'
    selection_flags_discovery:
        CommandLine|contains|all:
            - '/export'
            - '/cfg'
    selection_flags_configure:
        CommandLine|contains|all:
            - '/configure'
            - '/db'
    filter:
        SubjectUserName|endswith: '$'
    condition: selection_img and (1 of selection_flags_*) and not filter
falsepositives:
    - Legitimate administrative use
level: medium
```

File: rules/windows/process_creation/proc_creation_win_susp_secedit.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
+title: Potential suspicious activity using secedit.exe
+id: c2c76b77-32be-4d1f-82c9-7e544bdfe0eb
+status: experimental
+description: Detects potential suspicious behaviour using secedit.exe
+references:
+    - https://blueteamops.medium.com/secedit-and-i-know-it-595056dee53d
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_susp_secedit.yml:6]:
```suggestion
    - https://blueteamops.medium.com/secedit-and-i-know-it-595056dee53d
    - https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/secedit
```

File: rules/windows/process_creation/proc_creation_win_susp_secedit.yml [hunk 3]
```diff
@@ -0,0 +1,49 @@
+title: Potential suspicious activity using secedit.exe
+id: c2c76b77-32be-4d1f-82c9-7e544bdfe0eb
+status: experimental
+description: Detects potential suspicious behaviour using secedit.exe
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_susp_secedit.yml:4]:
```suggestion
description: Detects potential suspicious behaviour using secedit.exe. Such as exporting or modifying the security policy
```

**Comment by nasbench** [comment]
Thanks for the PR. What do you think about the `/import` flag? Can it be used in a suspicious manner to import a modified db/config?

**Comment by blueteam0ps** [comment]
> Thanks for the PR. What do you think about the `/import` flag? Can it be used in a suspicious manner to import a modified db/config?

Good call out for that. However, import just imports the ini template to the ese db. At that stage it doesn't make any impact to the system. The only way to apply the config that is in the ese is by using the /configure switch. That is already covered.

File: rules/windows/process_creation/proc_creation_win_susp_secedit.yml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    - attack.T1562
+    - attack.T1574.007
+    - attack.T1564.002
+    - attack.T1546.008
+    - attack.T1546.007
+    - attack.T1547.014
+    - attack.T1547.010
+    - attack.T1547.002
+    - attack.T1557
+    - attack.T1082
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_susp_secedit.yml:30]:
```suggestion
tags:
    - attack.discovery
    - attack.persistence
    - attack.defense_evasion
    - attack.credential_access
    - attack.privilege_escalation
    - attack.t1562.002
    - attack.t1547.001
    - attack.t1546.007
    - attack.t1505.005
    - attack.t1556.002
    - attack.t1562
    - attack.t1574.007
    - attack.t1564.002
    - attack.t1546.008
    - attack.t1546.007
    - attack.t1547.014
    - attack.t1547.010
    - attack.t1547.002
    - attack.t1557
    - attack.t1082
```

File: rules/windows/process_creation/proc_creation_win_susp_secedit.yml [hunk 5]
```diff
@@ -0,0 +1,51 @@
+title: Potential suspicious activity using secedit.exe
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_susp_secedit.yml:1]:
```suggestion
title: Potential Suspicious Activity Using SeCEdit
```

**Review by nasbench — COMMENTED** [review]
Approved after changes

File: rules/windows/process_creation/proc_creation_win_susp_secedit.yml [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+date: 2022/11/18
+tags:
+    - attack.discovery
+    - attack.persistence
+    - attack.defense_evasion
+    - attack.credential_access
+    - attack.privilege_escalation
+    - attack.t1562.002
+    - attack.t1547.001
+    - attack.t1546.007
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_susp_secedit.yml:18]:
```suggestion
```

**Review by nasbench — CHANGES_REQUESTED** [review]

**Review by nasbench — APPROVED** [review]

**Pull request merged**